### ISSUE TITLE
Find a way to display messages

### ISSUE THREAD

**Issue by menor** [resmio/widget#79]
Mainly error messages, I still have no idea about how to do it in Flux, but something like 'The server is not working, please try again later' or 'You need to select a time to be able to post a booking'

Need help from @fabiennepimenta

**Comment by fabiennepimenta** [comment]
Yes. No problem. I already made something about it for the iPad app. 

Basically, the errors will appear after the header , like this:
( if possible turn the object where the error is also in red)

<img width="339" alt="screen shot 2015-10-05 at 16 29 21" src="https://cloud.githubusercontent.com/assets/3285116/10283092/4a9670f6-6b7e-11e5-8cb9-fc54a33c254e.png">

**Comment by menor** [comment]
That looks great

**Comment by fabiennepimenta** [comment]
:+1:
